functions.php

```diff
@@ -102,6 +102,7 @@ function (): void {
         Api\Search::register_endpoint();
         Api\Settings::register_endpoint();
         Api\ENForm::register_endpoint();
+        Api\Covers::register_endpoint();
```

Comment:
This is an idea for another PR:
It seems to me that such an important piece of code like this one in charge of registering the API endpoints should be part of a separate class and not just a hook in the middle of the `functions.php` file.

Reply:
No comments here so I assume you disagree 😆  That's okay

Reply:
Sorry no I don't disagree haha! I don't have a strong opinion either way.